crates/bevy_text/src/font_atlas.rs

```diff
@@ -12,19 +12,27 @@ use crate::{GlyphAtlasLocation, TextError};
 
 /// Rasterized glyphs are cached, stored in, and retrieved from, a `FontAtlas`.
 ///
-/// A [`FontAtlasSet`](crate::FontAtlasSet) contains one or more `FontAtlas`es.
+/// A `FontAtlas` contains one or more textures, each of which contains one or more glyphs packed into them.
+///
+/// A [`FontAtlasSet`](crate::FontAtlasSet) contains a `FontAtlas` for each font size in the same font face.
+///
+/// For the same font face and font size, a glyph will be rasterized differently for different subpixel offsets.
+/// In practice, ranges of subpixel offsets are grouped into subpixel bins to limit the number of rasterized glyphs,
+/// providing a trade-off between visual quality and performance.
+/// A [`CacheKey`](cosmic_text::CacheKey) encodes all of the information of a subpixel-offset glyph.
```

Comment:
This sentence isn't clear to me. Why is a key encoding information? Wouldn't it be used to look it up?